add info about added fields (#9922)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/vectorcreation.rst

   * - **Offset lines**
     - ``OUTPUT``
     - [vector: line]

>      - Output line layer with offset features. Attributes will be copied to the corresponding output.
>        The original features are also copied.
>        The following attributes related to the input parameters will be added:
>        ``instance`` indexing the ``COUNT`` offset lines and ``offset``
>        designating the offset from the original line.

Reviewer: Let's group related topics (features vs fields)
```suggestion
     - Output line layer with the original and offset features.
       Attributes will be copied to the corresponding outputs.
       The following attributes related to the input parameters will be added:
       ``instance`` indexing the ``COUNT`` offset lines and ``offset``
       designating the offset distance from the original line.
```
